src/lib/sandbox/messenger.ts

```diff
@@ -47,6 +48,24 @@ export const onMessageFromWebWorker = (
     // web worker has finished initializing the script, and has another one to do
     // doing this postMessage back-and-forth so we don't have long running tasks
     initializedWorkerScript(winCtx, doc, msg[1] as number, msg[2] as string);
+  } else if (msgType === WorkerMessageType.ForwardWorkerAccessRequest) {
```

Comment:
Ideally, I would like to move this to `src/lib/atomics/sync-create-messenger-atomics.ts`. For now I can't because `createMessengerAtomics` doesn't have an access to worker instance so I'm unable to add additional messsage listener for it.